codes/rust/chapter_array_and_linkedlist/linked_list.rs

```diff
@@ -26,9 +26,9 @@ pub fn remove<T>(n0: &Rc<RefCell<ListNode<T>>>) {
     };
     // n0 -> P -> n1
     let P = n0.borrow_mut().next.take();
-    if let Some(node) = P {
-        let n1 = node.borrow_mut().next.take();
```

Comment:
please do not modify the variable names. They are consistent across different programming languages.

Reply:
oh, got you